account-management-api/docs/account-management-api.md

```diff
@@ -0,0 +1,240 @@
+| `replacementEmailAddress` | Required   | The user’s new email address. |
+| `otp`                     | Required   | The OTP that we sent to the user’s new email address and was entered by the user when requesting the update. |
+
+If the request is successful, the account management API returns this response:
+```
+204 No Content
+```
+
+### Delete a user’s account
+
+You can make a `POST` request to the `/delete-account` endpoint and delete a user’s account by providing the user’s email address.
+
+If successful, GOV.UK Notify sends an email to the user’s email address confirming the deletion of the user’s account.
+
+Example request:
+```
+POST /delete-account
+{
+ "email": "test@example.com"
+}
+```
+
+Request parameters: 
+| Parameter       | Required or optional | Description               |
+|-----------------|----------------------|---------------------------|
+| `email`         | Required             | The user's email address. |
+
+
+If the request is successful, the account management API returns this response:
+```
+204 No Content
+```
+
+### Getting a user’s selected MFA method 
+
+You can make a `POST` request to the `/mfa-method` endpoint to find out a user’s selected MFA method by providing the user’s email address.
+
+The only 2 MFA methods available are to: 
+* get an OPT sent by text message to your mobile phone 
+* use an authenticator app 
+
+Example request:
+```
+GET /mfa-method
+{
+ "email": "test@example.com"
+}
+```
+
+Request parameters: 
+| Parameter       | Required or optional | Description               |
+|-----------------|----------------------|---------------------------|
+| `email`         | Required             | The user's email address. |
+
+Example response for a successful request: 
+```
+{
+ "primaryMfaMethod": "AUTH_APP"
+}
+```
+
+Response parameters:
+| Parameter          | Description               |
+|--------------------|---------------------------|
+| `primaryMfaMethod` | The user’s primary MFA method when signing in to [GOV.UK One Login services at medium Cl.Cm protection level](https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/choose-the-level-of-authentication/#choose-the-level-of-authentication-for-your-service). <br><br> The 2 available methods are: <ul><li>`AUTH_APP` for users wanting to use an authenticator app</li><li>`SMS` for users wanting to receive OTPs on their mobile phone</li></ul> |
+
+### Update a user’s MFA method
+
+You can make a `POST` request to the `/update-mfa-method` endpoint to change a user’s MFA method. 
+
+When changing the MFA method from authenticator app to SMS, you must call the [`/send-otp-notification` endpoint](https://github.com/alphagov/di-authentication-api/edit/bau-draft-account-management-api-doc/account-management-api/docs/account-management-api.md#send-an-otp-notification-to-a-user) first to send an OTP to the user's mobile phone.
+
+If successful,
+* we remove the existing MFA method 
```

Comment:
Is there a missing bullet here? Presumably this request also adds the new MFA method?